content/docs/dvclive/api-reference/live/index.md

```diff
@@ -54,6 +55,11 @@ files.
   _metrics summary_ (usable by `dvc metrics`). The _summary_ will be located at
   `{path}.json`. _Default_: `True`.
 
+- `no_step` - If `True`, DVCLive will **not** generate
+  [_metrics logs_](/doc/dvclive/get-started#metrics-logs),
+  [_html report_](/doc/dvclive/dvclive-with-dvc#html-report) nor
+  [_checkpoints_](/doc/dvclive/dvclive-with-dvc#checkpoints)
```

Comment:
I do have one generic note on existing DVCLive docs writing style: It's preferable to avoid manual styling of text even when it is a feature in Markdown. Especially of link elements which already have CSS styling, but other italics and bolds should be pretty rare if any at all. When they're common (e.g. on every "not") it kind of defeats their purpose of applying special emphasis ("will/does/can not" is already emphatic). I also know italics can be used on certain key terms but maintaining this kind of rule manually is nearly impossible -- if needed it should be automated.

Reply:
p.s. I'm not saying we should un-do existing styling since it's already in though. But maybe consider stopping applying it and create an issue to remove it later if you agree.